### DISCUSSION TITLE
Colored vs white rect

### DISCUSSION THREAD

**Discussion by fileformat** [edent/SuperTinyIcons#122] [General]
I noticed that some logos (examples: twitter, stack overflow, instagram, skype) have been changed from the reference to be a white path on a colored rect.

Some others (examples: android, disqus, docker, npm) have kept the white rect & colored path.

I personally like the colored rect versions and have tweaked several of them accordingly for [myself](http://www.vectorlogo.zone/logos/visual.html#tile).  If you want these updates, let me know & I will minimize them & submit a PR.

**Comment by edent** [comment]
It mostly depends on how I was feeling when I created them :-)

Happy to take PRs.  Please note, I've changed the layout of the SVGs - this makes them more consistent and is easier to see what's changed in a PR.  New format is

```
<svg xmlns="http://www.w3.org/2000/svg"
aria-label="Icon Name" role="img"
viewBox="0 0 512 512"><rect
width="512" height="512"
rx="15%"
fill="#fff"/>...</svg>
```

There's no spaces at the end of the line, just a newline.

**Comment by 17jiangz1** [comment]
I think the iconset should be standardised to prefer white/official colours. At this point, the convention seems to be more leaning towards coloured backgrounds but is still quite varied.

**Comment by edent** [comment]
Yup. They're mostly using whatever the official logo uses.
